mandatory/2-function-creation.js

```diff
@@ -6,7 +6,10 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  return Number.isInteger(num) && num % 2 === 0 && num <= 100;
```

Comment:
Great job, I like that you managed to use logical operates in one single return statement, my only suggestion would be to add parenthesis to make it easier to read at first glance.

```javascript
function validate(num) {
  return (Number.isInteger(num) && (num % 2 === 0) && (num <= 100));
}
```

This just makes it easier to spot each of the logical operators and immediately identify all 3 conditions separately.